src/componantes/coins/coinRender.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Coins.css';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { detail } from '../redux/actions';
+
+export default function CoinRender({
+  List: {
+    id, icon, price, name,
+  },
+}) {
+  const CurrencyFormatter = (amount) => new Intl.NumberFormat('en-US', {
+    style: 'currency',
+    currency: 'USD',
+  }).format(amount);
+  const dispatch = useDispatch();
+  const clickhandler = (e) => {
+    dispatch(detail(e.target.id));
+  };
+  return (
+    <NavLink title={name} to="details" className="cards" id={id} onClick={clickhandler} key={id}>
+      <img className="icons" src={icon} id={id} alt={id} />
+      <div id={id}>{name}</div>
+      <div id={id}>{CurrencyFormatter(price)}</div>
+    </NavLink>
+  );
```

Comment:
- [ ] Good job so far :+1: Your site looks great and it has the correct functionality :fire: Kindly arrange your mobile design to match that of the template where the color of the cards are alternating.
- Your design:
![Screenshot from 2022-07-28 12-50-24](https://user-images.githubusercontent.com/85113567/181499522-ec4bdb0e-6e21-4052-b383-997d20a5799b.png)
- Required design:
![Screenshot from 2022-07-28 12-51-21](https://user-images.githubusercontent.com/85113567/181499569-dde1944f-e43b-45c2-a2bb-707736a017cb.png)